index.html

```diff
@@ -1,17 +1,130 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Capstone Project - Concert Page</title>
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="index.css" />
+    <link
+      href="https://fonts.googleapis.com/css?family=Lato"
+      rel="stylesheet"
+    />
+    <link href="https://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet" />
     <script src="index.js"></script>
-</head>
-<body>
-  <header>
-    
-  </header>
-    
-</body>
-</html>
\ No newline at end of file
+
+    <title>Capstone Project - Concert Page</title>
+  </head>
+  <body>
+    <header>
```

Comment:
Great job on using the right semantics here, the styling is good too but we have to follow the design guidelines 😓 

- [ ] Kindly make the top navbar dark and the one below it light colored as showed in the guidelines (for both pages) 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/204088098-f0d92af7-4e8b-4882-b35d-f7c4f5c4c1af.png)

## Guidelines:
![image](https://user-images.githubusercontent.com/50721479/204088107-3394f9d2-83bd-4e21-b48f-98bf9a5103c5.png)

Reply:
- [ ] This issue is well resolved :heavy_check_mark: